packages/orbit-tailwind-preset/src/presets/__tests__/configs.test.ts

```diff
@@ -4,7 +4,7 @@ import orbitPreset from "../..";
 
 describe("orbitPreset", () => {
   it("should match snapshot", () => {
-    const cfg = resolveConfig(orbitPreset());
+    const cfg = resolveConfig(orbitPreset({ disablePreflight: true }));
```

Comment:
Can we have another test with `orbitPreset()` as we had before?

Reply:
why? it's tested as with and without

Reply:
ok, we can, only for sake to have tested default behaviour